apps/dashboard-e2e/src/describer/fairness/IFairnessMetadata.ts

```diff
@@ -9,17 +9,20 @@ export interface IFairnessMetadata {
   defaultPerformanceMetric: string;
   fairnessMetrics: string[];
   defaultFairnessMetric: string;
+  defaultErrorMetric: string;
   numberOfModels: number;
   charts: string[];
 }
 
 export enum PredictionTypes {
   BinaryClassification = "binaryClassification",
+  BinaryClassificationWithError = "binaryClassificationWithError",
```

Comment:
It's necessary for the code later on to retrieve the right URL that has the error bars default on. While `checkErrorBars` is for the testing platform to know to test for the error bars

Reply:
What do you mean by 
> retrieve the right URL
?
Where is that happening? It really seems like this should just be a separate bool rather than making this enum more complicated.